turkce/catalla-kaz/kus/kus.md

```diff
@@ -0,0 +1,13 @@
+Kusmaya başladın.
+
+O ses kulaklarından hiç gitmediği için sürekli kusmaya devam ediyordun.
+
+Biraz duvarları yesem geçer diye düşündün.
+
+Ama ya geçmezse?
+
+Bir yol bulup ilerlemek de bir seçenekti.
+
+Ne yapacaksın?
+
+[Duvarları yeme başla?](https://github.com/udacity/create-your-own-adventure/blob/master/turkce/duvarlari-ye/duvarlari-ye.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.